c/include/libsbp/cpp/sbp_stdio.h

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (C) 2020 Swift Navigation Inc.
+ * Contact: Swift Navigation <dev@swiftnav.com>
+ *
+ * This source is subject to the license found in the file 'LICENSE' which must
+ * be be distributed together with this source. All other rights reserved.
+ *
+ * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+ * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+ */
+
+#ifndef SBP_CPP_STDIO_H
+#define SBP_CPP_STDIO_H
+
+#include <fstream>
+
+#include <libsbp/cpp/message_handler.h>
+
+namespace sbp {
+
```

Comment:
Following example of [state.h](https://github.com/swift-nav/libsbp/blob/master/c/include/libsbp/cpp/state.h) have implementation in header file too.

Reply:
My original intention was that all of the C++ wrappers would be header only. I think that's fine to continue here, especially since these are nice small classes.